### ISSUE TITLE
When training Centernet on my own dataset, the IOU during traing is always equal to 0.

### ISSUE THREAD

**Issue by ssli23** [open-mmlab/mmdetection#7857]
Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=1000 ] = 0.000
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=1000 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=1000 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=1000 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=1000 ] = -1.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=300 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=1000 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=1000 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=1000 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=1000 ] = -1.000

2022-04-28 16:11:27,814 - mmdet - INFO - Exp name: centernet_resnet18_dcnv2_140e_nih.py
2022-04-28 16:11:27,814 - mmdet - INFO - Epoch(val) [3][829]    bbox_mAP: 0.0000, bbox_mAP_50: 0.0000, bbox_mAP_75: 0.0000, bbox_mAP_s: 0.0000, bbox_mAP_m: 0.0000, bbox_mAP_l: -1.0000, bbox_mAP_copypaste: 0.000 0.000 0.000 0.000 0.000 -1.000

I do not think it's a data error problem because when I train with mask_rcnn, it's learning normally.

**Comment by RangiLyu** [comment]
Please provide more information such as the hyperparameters and the batch size you used during training.

**Comment by ssli23** [comment]
> Please provide more information such as the hyperparameters and the batch size you used during training.
batchsize = 16,
All other parameters are default, based on the centernet_resnet18_dcnv2_140e_coco.py

**Comment by RangiLyu** [comment]
The default config is trained based on 8GPUs. Try to decrease the learning rate.

**Comment by Linaom1214** [comment]
> > Please provide more information such as the hyperparameters and the batch size you used during training.
> > batchsize = 16,
> > All other parameters are default, based on the centernet_resnet18_dcnv2_140e_coco.py

how the solve the problem?

**Comment by soHardToHaveAName** [comment]
我遇到了同样的问题，loss迟迟不下降，AP总是0. 解决方法确实如 ‘RangiLyu’ 同学说的，需要修改学习率。

默认的配置文件是按照 `batchsize = 8gpu * 16img= 128` 来设置的learning rate。

把最后一句 `auto_scale_lr = dict(base_batch_size=128) ` 修改为 ` auto_scale_lr = dict(enable=True, base_batch_size=128) ` 即可自动调整学习率

**Comment by clearlove7-piggy** [comment]
> 
老哥，改了自动调整学习率了为啥训练时候AP还是0
